packages/ui/src/working-groups/components/ApplicationsList.tsx

```diff
@@ -47,7 +47,7 @@ const ApplicationListItem = ({ application, past }: { application: WorkingGroupA
     <ApplicationItemWrap past={past}>
       <ApplicationItemInfo>
         <ToggleableItemInfoTop>
-          <ApplicationID title={application.id}>ID: {application.id}</ApplicationID>
+          <ApplicationID title={application.id}>ID: {application.id.replace(/\D/g, '')}</ApplicationID>
```

Comment:
That's something that I have to think about. We could either:

*   use `runtimeId`
*   don't do that

The reason is that the ID is "weird" for applications (AFAIR it is group name + runtime id number).

Reply:
Then in OpeningListItem.tsx and UpcomingOpeningListItem.tsx this part should be fixed too